src/app/components/entityView/RowView.jsx

```diff
@@ -104,7 +104,11 @@ class View extends PureComponent {
       [ColumnKinds.status]: StatusView
     };
 
-    const isDisabled = !(this.canEditValue() && !lockStatus && kind !== ColumnKinds.status);
+    const isDisabled = !(
+      this.canEditValue() &&
+      !lockStatus &&
+      kind !== ColumnKinds.status
```

Comment:
Haben wir den Statuscolumn-Vergleich nicht schon zu `canEditValue` reingepackt, weil wir StatusSpalten eh nie ändern können?

Reply:
Habs jetzt in canUserChangeCell verschoben, denke da machts am meisten Sinn.